src/Util/PHP/Template/TestCaseMethod.tpl.dist

```diff
@@ -27,6 +27,10 @@ if ($composerAutoload) {
 
 function __phpunit_run_isolated_test()
 {
+    // if testcase open stdout and close it, the output will lost.
```

Comment:
Full dot at the end, but sentence is not started with upper case.
`will be lost`

Reply:
Fixed